.obs/dockerfile/Dockerfile

```diff
@@ -0,0 +1,61 @@
+# SPDX-License-Identifier: Apache-2.0
+# Define the names/tags of the container
+#!BuildTag: rancher/elemental-teal/5.3:latest
+#!BuildTag: rancher/elemental-teal/5.3:%VERSION%
+#!BuildTag: rancher/elemental-teal/5.3:%VERSION%-%RELEASE%
+#
+
+FROM suse/sle-micro-rancher/5.3:latest
```

Comment:
I wonder if we shouldn't make `5.3` a build config macro 🤔

Reply:
Probably yes, but I would leave it for the time we need it. Hard to foresee now how it actually should be used.